src/cypress-runner.js

```diff
@@ -51,9 +40,9 @@ const getCypressOpts = function (runCfg, suiteName) {
     throw new Error(`Could not find suite named '${suiteName}'; available suites='${suites}`);
   }
 
-  let cypressCfgFile = path.basename(runCfg.cypress.configFile);
-  if (!fs.existsSync(cypressCfgFile)) {
-    throw new Error(`Unable to locate the cypress config file. Looked for '${cypressCfgFile}'.`);
+  let cypressCfgFile = runCfg.cypress.configFile;
+  if (!fs.existsSync(getAbsolutePath(cypressCfgFile))) {
+    throw new Error(`Unable to locate the cypress config file. Looked for '${path.join(process.cwd(), cypressCfgFile)}'.`);
```

Comment:
Shouldn't that be:
```suggestion
    throw new Error(`Unable to locate the cypress config file. Looked for '${getAbsolutePath(cypressCfgFile)}'.`);
```
After all, that's what the if-statement was checking 🤔 Certainly possible that the absolute path provided had no config file at the end of the tunnel. The error would then report a potentially wrong path.